test/runner.c

```diff
@@ -210,6 +210,7 @@ int run_test(const char* test,
 #ifndef _WIN32
   /* Clean up stale socket from previous run. */
   remove(TEST_PIPENAME);
+  remove(TEST_PIPENAME_2);
```

Comment:
This not being here caused some issues where failed tests could leave things in a bad state.